packages/main/src/themes/SelectPopover.css

```diff
@@ -6,8 +6,4 @@
 .ui5-select-popover [ui5-option],
 .ui5-select-popover [ui5-option-custom] {
 	height: var(--_ui5_list_item_dropdown_base_height);
```

Comment:
this selector seems to be redundant too

Reply:
This is needed as the option's height is different when in dropdown according to VD spec. Moved it to OptionBase.css file as seems better.